roles/common/tasks/ntp.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install NTP
+  yum:
+    name: ntp
+    state: present
+    update_cache: true
+
+- name: Configure NTP
+  template:
```

Comment:
If this is a template, then the file should have the [`.j2`](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/template_module.html) (preferable), [`.jinja`](https://jinja.palletsprojects.com/en/3.0.x/templates/#template-file-extension), or [`jinja2`](https://marketplace.visualstudio.com/items?itemName=samuelcolvin.jinjahtml) extension at the end of the name and it should be parameterized, that is, some variables shoul be added to it.

Reply:
fixed (add params to /defaults/main.yaml and add /templates/ntp.conf.j2)

Reply:
But you didn't change the source template extension --> `src: ntp.conf`